software/winrm/src/main/java/org/apache/brooklyn/core/sensor/windows/WinRmCommandSensor.java

```diff
@@ -144,15 +144,7 @@ public T apply(String input) {
     public static String makeCommandExecutingInDirectory(String command, String executionDir, Entity entity) {
         String finalCommand = command;
         String execDir = executionDir;
-        if (Strings.isBlank(execDir)) {
-            // default to run dir
-            execDir = entity.getAttribute(BrooklynConfigKeys.RUN_DIR);
-            // if no run dir, default to home
-            if (Strings.isBlank(execDir)) {
-                execDir = "%USERPROFILE%";
-            }
-        }
-        if (!"~".equals(execDir)) {
+        if (Strings.isNonBlank(execDir) && !"~".equals(execDir)) {
```

Comment:
That's nice as a temporary patch around the fact that `RUN_DIR` is set to a UNIX-y path. What's the default folder we get on login? Is there a risk in executing in a protected folder (`C:\Windows\system32`) which will fail the code if it writes to the file system?
What if we keep the `%USERPROFILE%` default?
I thought this is executed as powershell, which makes the next line look like it would fail - how does it work then?